examples/smart_contract.ts

```diff
@@ -1,6 +1,6 @@
 import * as C from "capi/mod.ts"
 
-import { client } from "capi/frame/zombienet/examples/smart_contract/zombienet.toml/collator01/@v0.9.360/mod.ts"
```

Comment:
what is the reason for this change?

Reply:
capi --> localhost that is

Reply:
Changed the import map example scope mapping of `capi/` to `capi/mod.ts` bc it was causing the frustrating import rewrite suggestions. This meant that `capi/frame...` became invalid. That being said, closing this out.